resources/metadata/gctsExecuteABAPUnitTests.yaml

```
@@ -39,7 +39,7 @@ spec:
         mandatory: true
       - name: host
         type: string
-        description: Specifies the protocol and host adress, including the port. Please provide in the format '<protocol>://<host>:<port>'. Supported protocols are `http` and `https`.
+        description: Specifies the protocol and host adress, including the port. Please provide in the format `protocol://host:port`. Supported protocols are `http` and `https`.
```

I would recommend do change it to this to underline that `protocol` etc. are placeholders.
```suggestion
        description: Specifies the protocol and host adress, including the port. Please provide in the format `<protocol>://<host>:<port>`. Supported protocols are `http` and `https`.
```